cal/cal/templates/home_logged_in.html

```diff
@@ -26,6 +26,13 @@
         }
       });
     });
+    function runFullSync() {
+      if ("{{request.user.profile.first_time_authed}}" == "True") {
```

Comment:
You'll want a triple equals here (`===`). See [link](https://stackoverflow.com/questions/523643/difference-between-and-in-javascript)